test/e2e/framework/util.go

```diff
@@ -3920,6 +3920,33 @@ func WaitForControllerManagerUp() error {
 	return fmt.Errorf("waiting for controller-manager timed out")
 }
 
+// CheckForControllerManagerHealthy checks that the controller manager does not crash within "duration"
+func CheckForControllerManagerHealthy(duration time.Duration) error {
+	var PID string
+	cmd := "sudo docker ps | grep k8s_kube-controller-manager | cut -d ' ' -f 1"
+	for start := time.Now(); time.Since(start) < duration; time.Sleep(5 * time.Second) {
+		result, err := SSH(cmd, GetMasterHost()+":22", TestContext.Provider)
```

Comment:
is SSH still state of the art to access the master in e2e tests? I remember that we ported a lot of SSH calls to privileged pods long ago.

Reply:
It looks like all the related functionality (and other functionality that needs info from master) in this file is using SSH calls. For example RestartControllerManager(),  WaitForControllerManagerUp(), etc. Could you point me to an example that uses privileged pods for similar functionality? I am curious as to how that works.